frontend/public/components/masthead-toolbar.jsx

```diff
@@ -238,6 +258,27 @@ const MastheadToolbarContents = ({ consoleLinks, cv, isMastheadStacked }) => {
       });
     }
 
+    if (isTroubleshootingPanelEnabled) {
+      sections.push({
+        name: t('public~Troubleshooting'),
+        isSection: true,
+        actions: [
+          {
+            label: t('public~Signal Correlation'),
+            component: 'button',
+            callback: (e) => {
+              e.preventDefault();
+              dispatch(reduxAction('openTP'));
```

Comment:
Nit. For me, TP = Tech Preview

```suggestion
              dispatch(reduxAction('openTroubleshootingPanel'));
```